HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1675,18 +1675,18 @@ def self.get_heat_cap_eir_fflow_spec(compressor_type)
   # TODO
   #
   # @param cooling_system [TODO] TODO
-  # @param use_eer [TODO] TODO
+  # @param use_ceer [TODO] TODO
   # @return [nil]
-  def self.set_cool_curves_central_air_source(cooling_system, use_eer = false)
+  def self.set_cool_curves_dx(cooling_system, use_ceer = false)
     clg_ap = cooling_system.additional_properties
-    clg_ap.cool_rated_cfm_per_ton = get_cool_cfm_per_ton(cooling_system.compressor_type, use_eer)
+    clg_ap.cool_rated_cfm_per_ton = get_cool_cfm_per_ton(cooling_system.compressor_type, use_ceer)
     clg_ap.cool_capacity_ratios = get_cool_capacity_ratios(cooling_system)
     set_cool_c_d(cooling_system)
 
     case cooling_system.compressor_type
     when HPXML::HVACCompressorTypeSingleStage
       clg_ap.cool_cap_ft_spec, clg_ap.cool_eir_ft_spec = get_cool_cap_eir_ft_spec(cooling_system.compressor_type)
-      if not use_eer
+      if not use_ceer
```

Comment:
The `use_eer` variable name is a bit confusing, haven't thought of a better name, just replaced with "ceer" here.

Reply:
Why do we bother creating this variable and passing it in? It can result in inconsistent data where you pass in true but it's a central AC. Can't we just check the system types using e.g. `cooling_system.cooling_system_type`?

Reply:
Yes, I agreed that it looked confusing and unnecessary. I think the reason was that there're several methods that need that check to determine which logic to use, so it's set on the top and reused in the child methods. Let me try removing it and see how it looks like.

Reply:
You could create a hvac.rb method that everyone calls.